centreon/src/Core/Resources/Infrastructure/Repository/DbReadResourceRepository.php

```diff
@@ -125,63 +125,66 @@ public function findParentResourcesById(ResourceFilter $filter): array
         $collector = new StatementCollector();
         $this->sqlRequestTranslator->setConcordanceArray($this->resourceConcordances);
 
-        $request = 'SELECT SQL_CALC_FOUND_ROWS DISTINCT
-            1 AS REALTIME,
-            resources.resource_id,
-            resources.name,
-            resources.alias,
-            resources.address,
-            resources.id,
-            resources.internal_id,
-            resources.parent_id,
-            resources.parent_name,
-            parent_resource.resource_id AS `parent_resource_id`,
-            parent_resource.status AS `parent_status`,
-            parent_resource.alias AS `parent_alias`,
-            parent_resource.status_ordered AS `parent_status_ordered`,
-            parent_resource.address AS `parent_fqdn`,
-            severities.id AS `severity_id`,
-            severities.level AS `severity_level`,
-            severities.name AS `severity_name`,
-            severities.type AS `severity_type`,
-            severities.icon_id AS `severity_icon_id`,
-            resources.type,
-            resources.status,
-            resources.status_ordered,
-            resources.status_confirmed,
-            resources.in_downtime,
-            resources.acknowledged,
-            resources.passive_checks_enabled,
-            resources.active_checks_enabled,
-            resources.notifications_enabled,
-            resources.last_check,
-            resources.last_status_change,
-            resources.check_attempts,
-            resources.max_check_attempts,
-            resources.notes,
-            resources.notes_url,
-            resources.action_url,
-            resources.output,
-            resources.poller_id,
-            resources.has_graph,
-            instances.name AS `monitoring_server_name`,
-            resources.enabled,
-            resources.icon_id,
-            resources.severity_id
-        FROM `:dbstg`.`resources`
-        LEFT JOIN `:dbstg`.`resources` parent_resource
-            ON parent_resource.id = resources.parent_id
-            AND parent_resource.type = ' . self::RESOURCE_TYPE_HOST
-            . ' LEFT JOIN `:dbstg`.`severities`
-            ON `severities`.severity_id = `resources`.severity_id
-        LEFT JOIN `:dbstg`.`resources_tags` AS rtags
-            ON `rtags`.resource_id = `resources`.resource_id
-        INNER JOIN `:dbstg`.`instances`
-            ON `instances`.instance_id = `resources`.poller_id';
-
-        $request .= " WHERE resources.name NOT LIKE '\_Module\_%'
-            AND resources.parent_name NOT LIKE '\_Module\_BAM%'
-            AND resources.enabled = 1 AND resources.type != 3";
+        $resourceTypeHost = self::RESOURCE_TYPE_HOST;
+        $request = <<<"SQL"
```

Comment:
Info: "double quotes" are not required for heredoc
```suggestion
        $request = <<<SQL
```

Reply:
Yes I will fix for all the file because there are the same error at several locations ;)

Reply:
fixed ! :)